wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -1674,6 +1674,186 @@ def postprocessSystem(self, sys, data, args):
+                if range(len(self.types1)):
+                    for k in range(len(self.types1)):
+                        types1 = self.types1[k]
+                        types2 = self.types2[k]
+                        if (namem == types1 and namen == types2) or (namem == types2 and namen == types1):
+                            rij = self.rmin[k]
+                            wdij = self.emin[k]
+                            rij6 = rij**6
+                            acoef[m+num_lj_types*n] = math.sqrt(wdij) * rij6
+                            bcoef[m+num_lj_types*n] = 2 * wdij * rij6
+                            break
+                        else:
+                            rij = (lj_radii[m] + lj_radii[n])
+                            rij6 = rij**6
+                            wdij = math.sqrt(lj_depths[m] * lj_depths[n])
+                            acoef[m+num_lj_types*n] = math.sqrt(wdij) * rij6
+                            bcoef[m+num_lj_types*n] = 2 * wdij * rij6
+                else:
+                    rij = (lj_radii[m] + lj_radii[n])
+                    rij6 = rij**6
+                    wdij = math.sqrt(lj_depths[m] * lj_depths[n])
+                    acoef[m+num_lj_types*n] = math.sqrt(wdij) * rij6
+                    bcoef[m+num_lj_types*n] = 2 * wdij * rij6
+
+        self.force = mm.CustomNonbondedForce('(a/r6)^2-b/r6; r6=r2*r2*r2; r2=r^2; '
+                                'a=acoef(type1, type2); '
+                                'b=bcoef(type1, type2)')
+        self.force.addTabulatedFunction('acoef',
+                mm.Discrete2DFunction(num_lj_types, num_lj_types, acoef))
+        self.force.addTabulatedFunction('bcoef',
+                mm.Discrete2DFunction(num_lj_types, num_lj_types, bcoef))
+        self.force.addPerParticleParameter('type')
+        if (nonbondedMethod is PME or nonbondedMethod is Ewald or
+                nonbondedMethod is CutoffPeriodic):
+            self.force.setNonbondedMethod(mm.CustomNonbondedForce.CutoffPeriodic)
+        elif nonbondedMethod is NoCutoff:
+            self.force.setNonbondedMethod(mm.CustomNonbondedForce.NoCutoff)
+        elif nonbondedMethod is CutoffNonPeriodic:
+            self.force.setNonbondedMethod(mm.CustomNonbondedForce.CutoffNonPeriodic)
+        else:
+            raise AssertionError('Unrecognized nonbonded method [%s]' %
+                                 nonbondedMethod)
+        # Add the particles
+        for i in lj_indx_list:
+            self.force.addParticle((i-1,))
+        self.force.setUseLongRangeCorrection(True)
+        self.force.setCutoffDistance(nonbondedCutoff)
+
+        # Create customBondForce for the 1-4 scaled L-J interactions
+        lj14ScaleForce = mm.CustomBondForce('(a/r^6)^2-(b/r^6); r6=r2*r2*r2; r2=r^2;'
+                                            'a=acoef;'
+                                            'b=bcoef;')
+        lj14ScaleForce.addGlobalParameter('lj14scale', self.lj14scale)
```

Comment:
This is used for the 1-4 scaled L-J.
https://github.com/ChayaSt/openmm/blob/charmm36/devtools/forcefields/charmm/tests/test_ffconversion.py

Here's the output of that script:
```
pamed energies: [('HarmonicBondForce', 8.26018942073294), ('HarmonicAngleForce', 0.2714968886612339), ('PeriodicTorsionForce', 1.0799992422864986), ('HarmonicBondForce', 0.007913540341948437), ('NonbondedForce', -50.08503469213911), ('CustomNonbondedForce', -0.09783352740061724), ('CMMotionRemover', 0.0)]
omm eneriges [('HarmonicBondForce', 8.268102961074886), ('HarmonicAngleForce', 0.271496888687008), ('PeriodicTorsionForce', 1.0799992422864986), ('CustomTorsionForce', 0.0), ('NonbondedForce', -50.07130056020566), ('CustomNonbondedForce', -0.09783352740099871), ('CustomBondForce', -0.013734131933562961), ('CMMotionRemover', 0.0)]
``` 
If you look at the ```energy_decomposition_system```, you can see that the ```CustomBondForce``` computes the 1-4 scaled LJ. In ParmEd, ```NonbondedForce``` calculates that with the exceptions. But since we are not using ```NonbondedForce``` here, I needed to add the ```CustomBondForce```.